indra/llrender/llfontregistry.cpp

```diff
@@ -289,23 +289,76 @@ bool font_desc_init_from_xml(LLXMLNodePtr node, LLFontDescriptor& desc)
         {
             std::string font_file_name = child->getTextContents();
             std::string char_functor;
+            EFontHinting hinting = EFontHinting::FORCE_AUTOHINT;
+            S32 flags = 0;
 
             if (child->hasAttribute("functor"))
             {
                 child->getAttributeString("functor", char_functor);
             }
 
+            if (child->hasAttribute("font_hinting"))
+            {
+                std::string attr_hinting;
+                child->getAttributeString("font_hinting", attr_hinting);
+
+                if (attr_hinting == "default")
+                {
+                    hinting = EFontHinting::DEFAULT;
+                }
+                else if (attr_hinting == "force_auto")
+                {
+                    hinting = EFontHinting::FORCE_AUTOHINT;
+                }
+                else if (attr_hinting == "no_hinting")
+                {
+                    hinting = EFontHinting::NO_HINTING;
+                }
+            }
+
+            if (child->hasAttribute("font_hinting"))
```

Comment:
Should we do it twice?

Reply:
Probably a merge conflic, will remove it, thanks